default.nix

```diff
@@ -45,6 +45,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     hostname
     python2 time # coq-makefile timing tools
+    jbuilder # why is this not part of ocamlPackages and why isn't it called dune yet?
```

Comment:
What is the purpose of this comment? Here are answers to the questions.

`jbuilder` is no OCaml library; its behavior does not depend on the version of the OCaml compiler used to compile it; therefore, it does not belong to the `ocamlPackages` set.

It is indeed not renamed yet, probably by lack of manpower.

Reply:
Thanks. It was mostly an poor-man's inline comment. I'll remove it.